command/helper/config.go

```diff
@@ -221,3 +238,22 @@ func readConfigFile(path string) (*Config, error) {
 
 	return &config, nil
 }
+
+func parseDNSString(s string) (version string, name string) {
+	s = strings.Trim(s, "/")
+	fmt.Println(s)
```

Comment:
Please remove the leftover print

Reply:
cleaned up

Reply:
Resolving this conversation. @munna0908 don't forget to include commit msg

cc @zivkovicmilos